tensorboard_reducer/load.py

```diff
@@ -74,9 +74,6 @@ def load_tb_events(
         all_dirs_tags_list = sorted(
```

Comment:
I do not know what this sort should really do and if this sort really works with the `missing_tags_report` zip statement: `zip(input_dirs, all_dirs_tags_list)` below. Maybe a special sort key is necessary (sort after event file name), maybe the sort should be removed?

Reply:
Don't remember why I put it there either. All tests pass without it so seems safe to remove.